dotnet/src/IntegrationTests/IntegrationTests.csproj

```diff
@@ -63,7 +63,6 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
-    <PackageReference Include="System.Text.Json" />
```

Comment:
Removing this reference as adding this creates a conflict with `AI.Abstractions` latest version.

`
IntegrationTests.csproj : error NU1605:  IntegrationTests -> Microsoft.SemanticKernel.Connectors.Ollama -> Microsoft.Extensions.AI 9.0.0-preview.9.24525.1 -> System.Text.Json (>= 9.0.0-rc.2.24473.5) 
`